src/crate/crash/command.py

```diff
@@ -382,8 +382,8 @@ def _switch_format(self, fmt=None):
         return u'{0} is not a valid output format.\nUse one of: {1}'.format(fmt, ', '.join(self.OUTPUT_FORMATS))
 
     def _connect(self, server):
-        """ connect to the given server """
-        self.connection = connect(servers=server, error_trace=self.error_trace)
+        """ connect to the given server, e.g.: \connect localhost:4200 """
+        self.connection = connect(servers=server, timeout=2, error_trace=self.error_trace)
```

Comment:
pls document default timeout change at changes. is this really used at crate-python?